src/bidiMapper/utils/cdpServer.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { CdpServer } from './cdpServer';
+import { ServerBinding } from './iServer';
+import { mock, instance, verify } from 'ts-mockito';
+
+describe('CdpServer tests.', async () => {
+  it('given CdpServer, when `sendMessage` is called, then cdpBindings should be called with proper values', async () => {
```

Comment:
Minor: With Mocha's interface, I think `describe('CdpServer')` serves the same purpose of saying "given CdpServer" and then "given CdpServer" doesn't need to be repeated in the test case names.

Suggestion for mocha-compatible style:

```js
describe('CdpServer', async () => {
  it('calls cdpBindings with proper values when `sendMessage` is called', async () => {
```

Reply:
done